src/lib/build.ts

```diff
@@ -33,6 +31,12 @@ export interface BuildOption {
   executableChromium?: string;
 }
 
+declare global {
+  export interface Window {
+    coreViewer: CoreViewer;
+  }
+}
+
```

Comment:
Would you place global type definitions to `src/global.d.ts` rather than here, to make available to use `window.coreViewer` from any source?

Reply:
Yes. I tried to initially, but I misunderstood TypeScript modules. Addressed now.